presto-main/src/main/java/io/prestosql/execution/DynamicFilterConfig.java

```diff
@@ -37,12 +37,12 @@
     private boolean enableLargeDynamicFilters;
     private int serviceThreadCount = 2;
 
-    private int smallBroadcastMaxDistinctValuesPerDriver = 100;
-    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(10, KILOBYTE);
-    private int smallBroadcastRangeRowLimitPerDriver;
-    private int smallPartitionedMaxDistinctValuesPerDriver = 10;
+    private int smallBroadcastMaxDistinctValuesPerDriver = 200;
+    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(20, KILOBYTE);
+    private int smallBroadcastRangeRowLimitPerDriver = 5000;
```

Comment:
let's disable min/max collection due to: https://github.com/prestosql/presto/issues/5368

Reply:
done